### PR TITLE
CSS change in the nav-bar

### PR DESCRIPTION
**Pull request by codlocker**
Added a design  to the <a> tags of .nav-links class. This is just to make the UI look more awesome :D .

### CODE REVIEW

**Comment by ingorichter** [comment]
Thanks for the contribution. I can see the animation underneath the links in the navbar in Safari, but not in Chrome. I'm using Chrome 54 on OSX. Do you have a chance to check this on Chrome 54?

BTW: did you sign the contributor license agreement (CLA)?

**Comment by codlocker** [comment]
I am using Chrome 54 on Windows 7. The animation is working here. 
image

**Comment by codlocker** [comment]
I am using Chrome 54 on Windows 7. I am getting the animation. I will try it on an OSX machine if I get one.
chrome54

**Comment by codlocker** [comment]
Can you give me the link to contributor license agreement (CLA)?

**Comment by ficristo** [comment]
I saw you signed the CLA 👍 
Please check better the format of the code: always 4 spaces indentation, braces on the same line, new lines between classes, ...
A personal opinion (you can disregard it) but I would prefer to see the line inside the navbar and not outside.

**Comment by ingorichter** [comment]
@codlocker don't look any further. I loaded the page in an incognito session and it worked fine. One of the numerous chrome plugins seems to prevent that animation.

LGTM now.

**Comment by ingorichter** [comment]
Please fix the formatting issues as @ficristo mentioned earlier. :-)

**Comment by codlocker** [comment]
I have fixed the formatting issues and moved the animation to inside the nav-bar as suggested by @ficristo.

**Review by ficristo — APPROVED** [review]

File: css/brackets.io.css [hunk 1]
```diff
@@ -316,6 +316,36 @@ blockquote p {
     margin: 0 0 0 90px;
 }
 
+/* CSS For nav-bar bottom border*/
+@media screen and (min-width:890px)
+{
+
```
Comment (ingorichter) [inline css/brackets.io.css]: nit: can we remove that blank line?

**Comment by ingorichter** [comment]
LGTM. Waiting for the removal of the blank line and then I'll merge it.

**Comment by codlocker** [comment]
Done @ingorichter

**Comment by codlocker** [comment]
Done @ingorichter, @ficristo

File: css/brackets.io.css [hunk 2]
```diff
@@ -316,9 +316,8 @@ blockquote p {
     margin: 0 0 0 90px;
 }
 
-/* CSS For nav-bar bottom border*/
-@media screen and (min-width:890px)
-{    
+@media screen and (min-width:890px) {/* CSS For nav-bar border animation*/
+
```
Comment (ingorichter) [inline css/brackets.io.css]: I thought this was gone ;-)

Comment (codlocker) [reply css/brackets.io.css]: @ingorichter Do I have to do any more changes?

Comment (ficristo) [reply css/brackets.io.css]: yep, remove that empty line.

**Comment by ficristo** [comment]
The last nits and then I can merge (I promise).
Could you use the this style: `prop: value; /* comment */`
No space before the colon, one after, one after the semicolon, one space before and after the comment.
Thank you for your patience.

**Comment by ficristo** [comment]
Thank you.

**Pull request merged**